Sources/SourceKit/sourcekitd/SwiftSourceKitFramework.swift

```diff
@@ -42,7 +42,11 @@ final class SwiftSourceKitFramework {
 
   init(dylib path: AbsolutePath) throws {
     self.path = path
+    #if os(Windows)
+    self.dylib = try dlopen(path.pathString, mode: [])
```

Comment:
`dlopen` is supported on Windows? I think we should use [`LoadLibraryA`](https://docs.microsoft.com/en-us/windows/desktop/api/libloaderapi/nf-libloaderapi-loadlibrarya).